.claude/skills/ai-content-pipeline/Skill.md

```
@@ -8,469 +8,159 @@ dependencies: python>=3.10
 ```
 
-**Model Options:**
-```bash
-# Gemini 3 Pro (default, recommended) - highest quality
-./venv/Scripts/aicp.exe analyze-video -i video.mp4 -m gemini-3-pro
+## Available Models (47 Total)
 
-# Gemini 2.5 Flash - faster, cost-effective
-./venv/Scripts/aicp.exe analyze-video -i video.mp4 -m gemini-2.5-flash
+| Category | Count | Examples |
+|----------|-------|----------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)